akka-http-core/src/main/scala/akka/http/scaladsl/model/Multipart.scala

```diff
@@ -328,6 +328,9 @@ object Multipart {
   object FormData {
     def apply(parts: Multipart.FormData.BodyPart.Strict*): Multipart.FormData.Strict = Strict(parts.toVector)
     def apply(parts: Multipart.FormData.BodyPart*): Multipart.FormData = Multipart.FormData(Source(parts.toVector))
+    // SI-2991 workaround - two functions below. Remove when this issue is fixed
```

Comment:
That's going to be a long lasting FIXME :) - the ticket itself has been re-opened and if I correctly understood the comments there is only partial, not complete, improvement expected in Dotty